v1/pages/_app.tsx

```diff
@@ -1,11 +1,11 @@
-import { AppProps } from "next/app";
-import "../styles/globals.css";
-import "../styles/index.scss";
-import "@agility/plenum-ui/lib/tailwind.css";
-import "easymde/dist/easymde.min.css";
+import { AppProps } from "next/app"
```

Comment:
Do we have prettier enabled in this repo? I suggest we make all our repos consistent